mill-universal-packager/src/io/github/hoangmaihuy/mill/packager/universal/Archiver.scala

```diff
@@ -0,0 +1,179 @@
+package io.github.hoangmaihuy.mill.packager.universal
+
+import java.io._
+import java.io.InputStream
+import java.nio.file.attribute.FileTime
+import java.util.concurrent.TimeUnit
+
+import scala.util.Using
+
+import mill.api.Ctx
+import os.{Path, PermSet, SubPath}
+import org.apache.commons.compress.archivers.tar.{TarArchiveEntry, TarArchiveOutputStream}
+import org.apache.commons.compress.compressors.CompressorOutputStream
+
+import io.github.hoangmaihuy.mill.packager.permissions.OctalString
+
+object Archiver {
```

Comment:
Can you run `scalafmt` on this file?

Reply:
added the ScalafmtModule as the following and reformat the whole project:
```
object `mill-universal-packager` extends ScalaModule with CiReleaseModule with ScalafmtModule {
```